projects/completed-projects/http.md

```diff
@@ -0,0 +1,47 @@
+## Description
```

Comment:
This should probably be under projects/ instead of completed-projects/, we still need to deliver on Stage 3, and move from frozen to stable

Reply:
no problem, I'll move it

Reply:
ping @tedsuo 🙂